packages/clarity/contracts/taral/exporter-storage.clar

```diff
@@ -62,32 +77,42 @@
     (ok (map-set exporter-by-principal exporter exporter-id))
 )
 
-(define-public (add-order (id uint) (exporter-id uint) (order-id uint)) 
+(define-public (add-order (id uint) (exporter-id uint) (hash (buff 256)) (order-id uint)) 
     (ok 
         (map-insert orders {id: id,exporter-id: exporter-id}   
-            {order-id: order-id }     
+            {order-id: order-id, hash: hash }     
```

Comment:
O think particularly for orders, we do not need to keep the hash property here as this is just a map that links exporters to their orders. Apologies for any confusion 😁

Reply:
Makes sense, i'll remove the hash property from here.